docs/cookbook/limit-repeated-requests.md

```diff
@@ -12,30 +15,78 @@ import * as http from 'http';
 // 3p
 import { Config, createApp } from '@foal/core';
 import * as express from 'express';
-import * as rateLimit from 'express-rate-limit';
+import rateLimit from 'express-rate-limit';
```

Comment:
This import is incorrect, the value of `rateLimit` is `undefined` in this case.

See https://github.com/nfriedly/express-rate-limit/blob/master/lib/express-rate-limit.js#L147

Reply:
I fixed this line because I am using the typescript branch of the express-rate-limit package (https://github.com/nfriedly/express-rate-limit/tree/typescript) which will probably be merged into master in the future (https://github.com/nfriedly/express-rate-limit/issues/138).

With this, the import is fine. But maybe we should import it like it is the official release, I agree.

I will fix this in a minute.

Reply:
Ok thanks!